Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: develop test
+
```

Comment:
In your particular examples (and the one in this PR) I don't have many complaints since it's so simple, but I think the syntax is unintuitive at times and results in a giant mess of spaghetti (:spaghetti: 💻 ). Anyways, what you have here seems simple enough so no reason to hold this PR up.